repos/fdbt-site/cypress_tests/cypress/support/multiOperatorGroups.ts

```diff
@@ -11,16 +17,13 @@ export const addSingleMultiOperatorGroup = (name: string, addExtra: boolean) =>
     clickElementById('search-button');
     clickElementByText("Warrington's Own Buses - WBTR");
     if (addExtra) {
-        clickElementById('search-input').clear().type('Pil');
-        clickElementById('search-button');
-        clickElementByText('Pilkingtonbus - NWBT');
+        addExtraOperator();
     }
     clickElementById('operator-group-name').clear().type(name);
     continueButtonClick();
 };
 
-const checkCardBody = (card: Cypress.Chainable<JQuery<HTMLElement>>, eq: number) => {
-    const valuesToCompare = ['Blackpool Transport - BLAC', "Warrington's Own Buses - WBTR"];
+const checkCardBody = (card: Cypress.Chainable<JQuery<HTMLElement>>, eq: number, valuesToCompare: string[]) => {
```

Comment:
Could we use a more descriptive term than 'eq' for the 2nd argument to this function? What does eq represent?

Reply:
[80bf3bf](https://github.com/fares-data-build-tool/create-data/pull/650/commits/80bf3bf98060f7659161007957a68440bd3960d3)